internal/adminanalytics/users.go

```diff
@@ -165,12 +163,12 @@ func (f *Users) MonthlyActiveUsers(ctx context.Context) ([]*MonthlyActiveUsersRo
 	prevMonth := now.AddDate(0, -2, 0) // going back 2 months
 	from := time.Date(prevMonth.Year(), prevMonth.Month(), 1, 0, 0, 0, 0, now.Location()).Format(time.RFC3339)
 
-	nonActiveUserEvents := []*sqlf.Query{}
-	for _, name := range eventlogger.NonActiveUserEvents {
-		nonActiveUserEvents = append(nonActiveUserEvents, sqlf.Sprintf("%s", name))
+	defaultConds, err := getDefaultConds(f.Ctx, f.DB, f.Cache)
```

Comment:
I don't understand why we need to set `Ctx context.Context` on all of these stores when we can just pass it into the methods?

```suggestion
	defaultConds, err := getDefaultConds(ctx, f.DB, f.Cache)
```